mteb/tasks/STS/fin/FinParaSTS.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class FinParaSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="FinParaSTS",
+        dataset={
+            "path": "TurkuNLP/turku_paraphrase_corpus",
+            "revision": "e4428e399de70a21b8857464e76f0fe859cabe05",
+            "name": "classification",
+        },
+        description="Finnish paraphrase-based semantic similarity corpus",
+        reference="https://huggingface.co/datasets/TurkuNLP/turku_paraphrase_corpus",
+        type="STS",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["fin-Latn"],
+        main_score="cosine_spearman",
+        date=("2017-01-01", "2021-12-31"),
+        form=["written"],
+        domains=["News", "Subtitles"],
+        task_subtypes=None,
+        license="cc-by-sa-4.0",
+        socioeconomic_status="high",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+        @inproceedings{kanerva-etal-2021-finnish,
+            title = "{F}innish Paraphrase Corpus",
+            author = {Kanerva, Jenna  and
+            Ginter, Filip  and
+            Chang, Li-Hsin  and
+            Rastas, Iiro  and
+            Skantsi, Valtteri  and
+            Kilpel{\"a}inen, Jemina  and
+            Kupari, Hanna-Mari  and
+            Saarni, Jenna  and
+            Sev{\'o}n, Maija  and
+            Tarkka, Otto},
+            editor = "Dobnik, Simon  and
+            {\O}vrelid, Lilja",
+            booktitle = "Proceedings of the 23rd Nordic Conference on Computational Linguistics (NoDaLiDa)",
+            month = may # " 31--2 " # jun,
+            year = "2021",
+            address = "Reykjavik, Iceland (Online)",
+            publisher = {Link{\"o}ping University Electronic Press, Sweden},
+            url = "https://aclanthology.org/2021.nodalida-main.29",
+            pages = "288--298",
+            abstract = "In this paper, we introduce the first fully manually annotated paraphrase corpus for Finnish containing 53,572 paraphrase pairs harvested from alternative subtitles and news headings. Out of all paraphrase pairs in our corpus 98{\%} are manually classified to be paraphrases at least in their given context, if not in all contexts. Additionally, we establish a manual candidate selection method and demonstrate its feasibility in high quality paraphrase selection in terms of both cost and quality.",
+        }        
+        """,
+        n_samples={"test": 1000, "validation": 1000},
```

Comment:
nit: use a variable `n_samples=1000`

Reply:
pydantic throws an error, as this must be a dictionary. will keep it as is :)

Reply:
Oh sorry. I meant something like this, and defining `N_SAMPLES=1000` around line 6 or 7.
```suggestion
        n_samples={"test": N_SAMPLES, "validation": N_SAMPLES},
```